specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/vmware.json

```diff
@@ -0,0 +1,1890 @@
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/hcxEnterpriseSiteName"
+          },
+          {
+            "name": "hcxEnterpriseSite",
+            "description": "The HCX Enterprise Site",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful update",
+            "schema": {
+              "$ref": "#/definitions/HcxEnterpriseSite"
+            }
+          },
+          "201": {
+            "description": "successful create",
+            "schema": {
+              "$ref": "#/definitions/HcxEnterpriseSite"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "HcxEnterpriseSites_CreateOrUpdate": {
+            "$ref": "./examples/HcxEnterpriseSites_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "HcxEnterpriseSites_Delete",
+        "tags": [
+          "HcxEnterpriseSites"
+        ],
+        "summary": "Delete an HCX Enterprise Site in a private cloud",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "#/parameters/hcxEnterpriseSiteName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The delete has succeeded"
+          },
+          "204": {
+            "description": "No Content. No HCX Enterprise Site to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "HcxEnterpriseSites_Delete": {
+            "$ref": "./examples/HcxEnterpriseSites_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/authorizations": {
```

Comment:
I'm not worried. I'd be okay to change it to `expressRouteAuthorizations` or just leave it. I'll ask our team how difficult it would be to change right now.

Reply:
After discussion, we decided to make the change. I'll update this API spec.

Reply:
After even further discussion, we are going to leave this as is. We appreciate the suggestion.